content/pt-br/docs/tasks/access-application-cluster/access-cluster-services.md

```diff
@@ -0,0 +1,124 @@
+
+Se você não especificou um nome para a porta, não é necessário especificar *nome_da_porta* na URL. Você também pode usar o número da porta no lugar do *nome_da_porta* para portas nomeadas e não nomeadas.
+
+Por padrão, o servidor da API usa um proxy para o seu serviço através de HTTP. Para usar HTTPS, adicione o prefixo `https:` ao nome do serviço:
+`http://<endereço_do_mestre_do_kubernetes>/api/v1/namespaces/<nome_do_namespace>/services/<nome_do_serviço>/proxy`
+
+Os formatos suportados para o segmento `<nome_do_serviço>` da URL são:
+
+* `<nome_do_serviço>` - usa um proxy para a porta padrão ou não nomeada usando http
+* `<nome_do_serviço>:<nome_da_porta>` - usa um proxy para a porta nomeada ou número da porta especificado usando http
+* `https:<nome_do_serviço>:` - usa um proxy para a porta padrão ou não nomeada usando https (observe o dois-pontos no final)
+* `https:<nome_do_serviço>:<nome_da_porta>` - usa um proxy para a porta nomeada ou número da porta especificado usando https
+
+##### Exemplos
+
+* Para acessar o _endpoint_ de serviço Elasticsearch `_search?q=user:kimchy`, você usaria:
+
+  ```
+  http://192.0.2.1/api/v1/namespaces/kube-system/services/elasticsearch-logging/proxy/_search?q=user:kimchy
+  ```
+  
+* Para acessar as informações de integridade do cluster Elasticsearch `_cluster/health?pretty=true`, você usaria:
+
+  ```
+  https://192.0.2.1/api/v1/namespaces/kube-system/services/elasticsearch-logging/proxy/_cluster/health?pretty=true
+  ```
+
+  As informações de integridade são semelhantes a estas:
+  
+  ```json
+    {
+      "cluster_name" : "kubernetes_logging",
+      "status" : "yellow",
+      "timed_out" : false,
+      "number_of_nodes" : 1,
+      "number_of_data_nodes" : 1,
+      "active_primary_shards" : 5,
+      "active_shards" : 5,
+      "relocating_shards" : 0,
+      "initializing_shards" : 0,
+      "unassigned_shards" : 5
+    }
+    ```
+    
+* Para acessar as informações de integridade do serviço Elasticsearch `_cluster/health?pretty=true`, você usaria:
+
+  ```
+  https://192.0.2.1/api/v1/namespaces/kube-system/services/https:elasticsearch-logging:/proxy/_cluster/health?pretty=true
+  ```
+
+#### Usando navegadores da web para acessar serviços em execução no cluster
+
+Você pode conseguir de colocar um URL de proxy do servidor da API na barra de endereço de um navegador. No entanto:
+
+  - Os navegadores da web geralmente não podem passar tokens, portanto, você pode precisar usar autenticação básica (senha). O servidor da API pode ser configurado para aceitar autenticação básica, mas o seu cluster pode não estar configurado para aceitar autenticação básica.
+  - Alguns aplicativos da web podem não funcionar, principalmente aqueles com javascript do lado do cliente que constroem URLs com um mecanismo que não está ciente do prefixo do caminho do proxy.
```

Comment:
You used "aplicações" before (line 33) and I liked it. 
"Algumas aplicações da web" instead of "Alguns aplicativos da web"